cli/paas/client.go

```diff
@@ -75,13 +79,24 @@ func (c *CarrierClient) Apps() error {
 		return errors.Wrap(err, "failed to list apps")
 	}
 
-	msg := c.ui.Normal().WithTable("Name", "Status", "Route")
+	msg := c.ui.Normal().WithTable("Name", "Status", "Routes")
 
 	for _, app := range apps {
-		msg = msg.WithTableRow(app.Name)
+		status, err := c.kubeClient.StatefulSetStatus(
+			c.config.EiriniWorkloadsNamespace,
+			fmt.Sprintf("cloudfoundry.org/guid=%s", app.Name))
+		if err != nil {
+			return errors.Wrap(err, "failed to get app status")
+		}
+
+		routes, err := c.kubeClient.ListIngressRoutes(
+			c.config.EiriniWorkloadsNamespace,
+			app.Name)
```

Comment:
Here we seem to ignore the possible error completely ?
IOW neither showing in the table, nor abort ?

Reply:
That's my mistake, thanks!